ibis/backends/postgres/__init__.py

```diff
@@ -225,11 +225,10 @@ def do_connect(
         Examples
         --------
         >>> import os
-        >>> import getpass
         >>> import ibis
         >>> host = os.environ.get("IBIS_TEST_POSTGRES_HOST", "localhost")
-        >>> user = os.environ.get("IBIS_TEST_POSTGRES_USER", getpass.getuser())
-        >>> password = os.environ.get("IBIS_TEST_POSTGRES_PASSWORD")
+        >>> user = os.environ.get("IBIS_TEST_POSTGRES_USER", "postgres")
```

Comment:
This is me being a nitpicker but do you believe we can replace `IBIS_TEST_*` prefixes by their common postgres version ?
- IBIS_TEST_POSTGRES_HOST => PGHOST
- IBIS_TEST_POSTGRES_USER => PGUSER
- IBIS_TEST_POSTGRES_PASSWORD =>  PGPASSWORD
- IBIS_TEST_POSTGRES_DATABASE =>  PGDATABASE
Saw this in other places as well maybe I can replace it all ?

Reply:
I think so, but let's do that in a follow up!